scripts/remove

```diff
@@ -12,106 +12,73 @@ source /usr/share/yunohost/helpers
 
 # Remove the dedicated systemd config
 ynh_remove_systemd_config
 
 #=================================================
 # REMOVE DEPENDENCIES
 #=================================================
-ynh_print_info "Removing dependencies"
+ynh_script_progression --message="Removing dependencies..." --weight=1
 
 # Remove metapackage and its dependencies
 ynh_remove_app_dependencies
-apt-get remove -y jellyfin-web jellyfin-server jellyfin-ffmpeg
+
 #=================================================
 # REMOVE APP MAIN DIR
 #=================================================
-ynh_print_info "Removing app main directory"
+ynh_script_progression --message="Removing app main directory..." --weight=12
 
 # Remove the app directory securely
-ynh_secure_remove "$final_path"
-ynh_secure_remove "/etc/jellyfin"
-ynh_secure_remove "/var/lib/jellyfin"
+ynh_secure_remove --file="$final_path"
+ynh_secure_remove --file="/etc/jellyfin"
+ynh_secure_remove --file="/var/lib/jellyfin"
 
 #=================================================
 # REMOVE NGINX CONFIGURATION
 #=================================================
-ynh_print_info "Removing nginx web server configuration"
+ynh_script_progression --message="Removing nginx web server configuration..." --weight=1
 
 # Remove the dedicated nginx config
 ynh_remove_nginx_config
 
-#=================================================
-# REMOVE LOGROTATE CONFIGURATION
-#=================================================
-ynh_print_info "Removing logrotate configuration"
-
-# Remove the app-specific logrotate config
-ynh_remove_logrotate
-
-#=================================================
-# CLOSE A PORT
-#=================================================
-
-if yunohost firewall list | grep -q "\- $port$"
-then
-	ynh_print_info "Closing port $port"
-	ynh_exec_warn_less yunohost firewall disallow TCP "$port"
-fi
-
-#=================================================
-# SPECIFIC REMOVE
-#=================================================
-# REMOVE THE CRON FILE
-#=================================================
-
-# Remove a cron file
-ynh_secure_remove "/etc/cron.d/$app"
-
-# Remove a directory securely
-ynh_secure_remove "/etc/$app/"
-
-# Remove the log files
-ynh_secure_remove "/var/log/$app/"
-
 #=================================================
 # GENERIC FINALIZATION
 #=================================================
 # REMOVE DEDICATED USER
 #=================================================
-ynh_print_info "Removing the dedicated system user"
+ynh_script_progression --message="Removing the dedicated system user..." --weight=2
 
 # Delete a system user
-ynh_system_user_delete "$app"
+ynh_system_user_delete --username=$app
 
 #=================================================
 # END OF SCRIPT
 #=================================================
 
-ynh_print_info "Removal of $app completed"
+ynh_script_progression --message="Removal of $app completed" --time --last
```

Comment:
```suggestion
ynh_script_progression --message="Removal of $app completed" --last
```

Reply:
I forgot to check out the linter... thanks :)